docs/provisioner/08-06-upgrading-shoots.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Upgrade shoots
+type: Tutorials
+---
+
+This tutorial shows how to upgrade Gardener shoots for Kyma Runtimes.
+
+## Steps
+
+> **NOTE:** To access the Runtime Provisioner, forward the port on which the GraphQL Server is listening.
+
+To upgrade a Gardener Shoot used to host the Runtime of a given ID, make a call to the Runtime Provisioner with a **tenant** header using a mutation like this:  
```

Comment:
```suggestion
To upgrade a Gardener Shoot used to host the Runtime of a given ID, make a call to the Runtime Provisioner with a **tenant** header using such a mutation:  
```

Reply:
As discussed, let's stick to `like this` :)